devenv/scripts/push_addon_image.sh

```diff
@@ -0,0 +1,25 @@
+# Get cluster information for az aks command invoke
```

Comment:
should this script be named `push_image`? From this project's perspective it isn't really an addon

Reply:
fair point!